widget/embedded/src/store/slices/data.ts

```diff
@@ -207,7 +207,10 @@ export const createDataSlice: StateCreator<
   // Actions
   fetch: async () => {
     try {
-      const response = await sdk().getAllMetadata();
+      const { enableCentralizedSwappers } = get().config;
+      const response = await sdk().getAllMetadata({
+        enableCentralizedSwappers,
+      });
```

Comment:
I see that we passed this parameter correctly to meta endpoint. ‌But I can't see 'XO Swap' in widget setting. Maybe we have a ui filter there on swapper type.